recipes/symstore/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "symstore" %}
+{% set version = "0.2.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f49ac8031b7572b5f3bcac3a8296d498fcd2a410225b3886f1c6b90b21c5bbf0"
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - symstore=symstore.command_line:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - symstore
+  commands:
+    - symstore --help
+
+about:
+  home: "https://github.com/elmirjagudin/symstore"
+  license: BSD
```

Which BSD licence should this be? The most common is:
```suggestion
  license: BSD-3-Clause
  license_family: BSD
  license_file: LICENSE
```
Also they all require the text of the license to be included so you should add a copy to your GitHub repository. You can then tag a new version or include it in the recipe directory so conda can pick it up.

---

I made a new release to pypi which contains the LICENSE file. I also decided to go with MIT license, after a closer review of license text :)